Cargo.toml

```diff
@@ -6,21 +6,25 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
```

Comment:
Sorted means less merge conflicts. Don't care that much. I have a shortcut to sort any set of lines so I tend to sort a lot. Especially in code. I don't expect others to do it.

Reply:
Good point, I agree. Out of curiosity do you use an automated command / plugin for this ?

Reply:
Maybe there is a way to include this in the `cargo fmt` linting ?

Reply:
```lua
vim.keymap.set("v", "<leader>co", ":sort<cr>", { noremap = true, silent = true, desc = "sort the selected lines" })
vim.keymap.set("n", "<leader>cb", "mzvi{:'<,'>sort<cr>`z", { noremap = true, silent = true, desc = "sort lines in {}" })
```